snapshots/windows/windows.go

```diff
@@ -60,8 +59,8 @@ func init() {
 const (
 	// Label to specify that we should make a scratch space for a UtilityVM.
 	uvmScratchLabel = "containerd.io/snapshot/io.microsoft.vm.storage.scratch"
-	// Label to control a containers scratch space size (sandbox.vhdx).
-	rootfsSizeLabel = "containerd.io/snapshot/io.microsoft.container.storage.rootfs.size-gb"
+	// Label to control a containers scratch space size in bytes (sandbox.vhdx).
+	rootfsSizeLabelBytes = "containerd.io/snapshot/io.microsoft.container.storage.rootfs.size-bytes"
```

Comment:
It seems kind of awkward to have something named like `io.microsoft...` in upstream containerd code. If we are changing label names, maybe we can make this more specific to containerd instead of Microsoft?

Reply:
That's a good point. Was curious how the device mapper snapshotter got its size to use, but it's just from its plugin section in the main config file. Not sure if anyone would want to configure that at runtime for any containers, but maybe its worth making this a generic `containerd.io/snapshot/rootfs.size-bytes` (or similar name) like you say.

Reply:
I moved this to a generic label under the snapshots package that snapshotters can choose to respect/implement as well as restored the original label we were using

Reply:
Sorry, I didn't necessarily mean it has to be common across all snapshotters, I just didn't like the "microsoft" naming. Something like "io.containerd.windows..." would be fine IMO.

Reply:
Is there precedent for label naming amongst other snapshotters?

Reply:
Some examples for snapshotter specific labels:
Overlay -> containerd.io/snapshot/overlay.upperdir
devmapper -> containerd.io/snapshot/devmapper/fstype

So for the most part it seems containerd.io/snapshot/{name_of_snapshotter}*

Reply:
Maybe `containerd.io/snapshot/windows/rootfs-sizebytes`, unless we expect to have multiple `rootfs.foo` values? Either way is okay though. Do you want to update the other label here as well to the new format?

Reply:
Given we were going to keep the other label to not introduce a breaking change, wouldn't changing the string be doing the same? It would keep the functionality (being giving the client a way to specify it in bytes) but they'd still need to change their script/code that uses it to this new name.

Reply:
Yeah if we change any label names then we should try to maintain backcompat. Just seems weird to have two different styles in the label names.

Reply:
You could also just leave all the labels the same for this PR to get the core piece of supporting the CRI field in, and worry about this later?

Reply:
I'd prefer to deprecate the old label and change to a new naming here so we don't need to go back and change after the fact